### DISCUSSION TITLE
How Jsbsim models the "+" shaped tail fins of missiles and rockets.

### DISCUSSION THREAD

**Discussion by ASh-en** [JSBSim-Team/jsbsim#1060] [Support requests]
Hello everyone, I would like to confirm whether \<htailarea>,  \<htailarm> , \<vtailarea>, \<vtailarm>  can also model the "+" tail fins of missiles and rockets. and also how to model aircraft with "+" shaped wings? And how to establish the relationship between $δ_x$, $δ_y$, $δ_z$ and the attitude of the aircraft?

**Comment by seanmcleod** [comment]
First off, just in case there are some potential assumptions. JSBSim isn't like Datcom or CFD which take in a geometric description of the aircraft and generate a flight model from that.

At the end of the day, as a 6DOF simulator, in terms of aerodynamics, JSBSim allows you to specify your own set of forces and moments and will then use them to calculate the net forces and moments and integrate them etc.

Take a look at the aerodynamics section of the aircraft models in the JSBSim repo.

Now in terms of things like `<htailarea>` and `<htailarm>` if you search for them in the JSBSim source code

https://github.com/search?q=repo%3AJSBSim-Team%2Fjsbsim+vbarh&type=code

You'll see that they are only used to generate metrics, which your functions in the forces and moments section of the aerodynamics section can make use of if you want to.

https://github.com/JSBSim-Team/jsbsim/blob/b9bfc2350e0e00e53246b65979cc18df2922a88d/src/models/FGAircraft.cpp#L154-L162

https://github.com/JSBSim-Team/jsbsim/blob/b9bfc2350e0e00e53246b65979cc18df2922a88d/src/models/FGAircraft.cpp#L181-L188

So in terms of modelling a specific missile or rocket, all you need to do is specify the relevant forces and moments in the aerodynamics section.
